src/components/Creatbook.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function Creatbook() {
+  return (
+    <div>Creatbook</div>
+  );
+}
```

Comment:
- [ ] @addisalemseifu Kindly add the form to add a new book here, I can see you added it in the booklist component, kindly move it here, also change the categories section to the author section as shown in the suggested code by @topeogunleye.